utils/src/test/java/org/apache/cloudstack/utils/mailing/SMTPMailSenderTest.java

```diff
@@ -0,0 +1,618 @@
+/*
+ * Copyright 2021 The Apache Software Foundation.
```

Comment:
I did not pay attention to these source code headers with Copyright but I now realize that you have been using this pattern on your PRs. Can you please change them all to fit the [Source File Headers for Code Developed at the ASF](https://www.apache.org/legal/src-headers.html#headers)?

Each source file should include the following license header -- **note that there should be no copyright notice in the header**:
```
Licensed to the Apache Software Foundation (ASF) under one
or more contributor license agreements.  See the NOTICE file
distributed with this work for additional information
regarding copyright ownership.  The ASF licenses this file
to you under the Apache License, Version 2.0 (the
"License"); you may not use this file except in compliance
with the License.  You may obtain a copy of the License at

  http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing,
software distributed under the License is distributed on an
"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
KIND, either express or implied.  See the License for the
specific language governing permissions and limitations
under the License.
```

Apache also makes it is very clear that such protocol (not adding Copyright) must be applied to Apache source code (e.g. CloudStack) but it does not apply to developers outside the ASF who are applying the Apache License to their work.

Reply:
ah, never thought it was a problem, we have loads of files with notices like this. Do we need a clean up action @GabrielBrascher ?

Reply:
@DaanHoogland that's a good question.

This header difference is a bit confusing especially due to the fact that Apache license 2.0 adds the Copyright, but at the same time, the documentation regarding Apache's source code do say to **not add** the copyright.

I think that @GutoVeronezi is used to work with https://www.apache.org/licenses/LICENSE-2.0 and then added it on his PRs. I have nothing against it; however, Apache is very clear about how to proceed in terms of [Source File Headers for Code Developed at the ASF](https://www.apache.org/legal/src-headers.html#headers), which I think fits on any CloudStack source header, regardless of who donated the contribution to the Apache

Reply:
Also, Apache refers to that document as:

> This document describes how Apache committers and PMC members should handle source file licensing and copyright notices.
It does not apply to developers outside the ASF who are applying the Apache License to their work.

The code is merged always by a **committer** that are adding a **contributor** code. Therefore, my understanding is that either @GutoVeronezi changes his code or then he leaves as it is but on the merge process the committer/PMC doing it should follow the Apache legal guidelines by adding a commit removing any Copyright on headers.

Reply:
@GabrielBrascher I changed my projects, so from now on they will contain ASF's header. Do I need to change all my open PRs to change it or leave them to be changed when they are merged?

Reply:
Thanks for the update @GutoVeronezi.

If possible to change them it would be nice, otherewise I would ask to add an observation regarding the source code header (if there is one of the PRs still open that we have not discussed the header yet).

One easy way of updating it (via UI) is with review, anyone (including you) can add code changes via reviews and them can be added to the PRs commits.

I will propose such changes, if you want you can commit them or ignore.